core/sqf/conf/log4cxx.trafodion.masterexe.config

```diff
@@ -43,19 +38,10 @@ log4j.appender.mxoAppender.addPid=false
 log4j.appender.mxoAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.mxoAppender.layout.ConversionPattern=%d, %p, %c, %m%n
 
+log4j.additive.mxoAppender=false 
+
 #MXOSRVR
 log4j.logger.MXOSRVR=ERROR, mxoAppender
 
 # SQL
-# uncomment the following line if you want to enable debug logging
-#log4j.logger.SQL.EXE=DEBUG
-log4j.logger.SQL=INFO
-log4j.logger.SQL.COMP=ERROR
-log4j.logger.SQL.EXE.Memory=INFO
-log4j.logger.SQL.ESP=ERROR
-log4j.logger.SQL.PRIVMGR=INFO
-#
-log4j.logger.SQL.HDFS.JniTop=FATAL
-log4j.logger.SQL.HDFS.SeqFileReader=FATAL
-log4j.logger.SQL.HDFS.SeqFileWriter=FATAL
-log4j.logger.SQL.HBase=FATAL
+log4j.logger.SQL=INFO, mxoAppender
```

Comment:
How come the granularity has changed ? Sometimes we want to log only Hbase  or Hdfs messages. Having the different levels helps. Plus if we have to make the default INFO  our code needs to change  to not use INFO in sqlmxevents APIs to log debugging type of messages which it does today. Those are not meant for the end user.